db/document.go

```diff
@@ -661,7 +661,38 @@ func HasUserXattrChanged(userXattr []byte, prevUserXattrHash string) bool {
 	return userXattrCrc32cHash(userXattr) != prevUserXattrHash
 }
 
-// SyncData.IsSGWrite - used during feed-based import
+// CVOutdated returns true if the provided CV does not match _sync.rev.ver and _sync.rev.src. The caller is responsible for testing if the values are non-empty.
+// This compares _vv.ver and _vv.src to _sync.rev.ver and _sync.rev.src
+func (s *SyncData) CVOutdated(cvSourceID string, cvVersion uint64) bool {
```

Comment:
```suggestion
func (s *SyncData) CVEqual(cvSourceID string, cvVersion uint64) bool {
```
I feel like calling this CVEqual is more straightforward (but would need to flip the return values from true/false), as "Outdated" suggests to me a "greater than" comparison, as opposed to matching.